packages/runtime/src/components/builtin/Button/Button.tsx

```diff
@@ -27,6 +26,7 @@ import {
   useResponsiveWidth,
 } from '../../utils/responsive-style'
 import { cx } from '@emotion/css'
+import { LinkValue } from '@makeswift/prop-controllers'
```

Comment:
We should decouple the value from the data, or is this the type for the value?

Reply:
I only copy paste the `LinkValue` and haven't updated the type yet.